CSS/style.css

```diff
@@ -0,0 +1,36 @@
+body {
+  max-width: 960px;
+  margin: 0 auto;
+  color: white;
+  background-color: black;
+}
+
+header {
+  text-align: center;
+}
+
+main {
+
+}
+
+footer {
+  text-align: center;
+}
+
+img {
+  /*pictures are getting stretched out in a bad way right now*/
```

Comment:
I mentioned this briefly in lecture... it makes life easier if you manually handle the resizing of images and then you can work with a fixed or percentage-based size for the image elements here.